cuda_phase-retrieval/phase_algo.hpp

```diff
@@ -464,14 +512,23 @@ __global__ void satisfy_fourier(cufftDoubleComplex *y_hat, cufftDoubleComplex *x
     {
         mag_comp.x = ptrMag[idx];
         mag_comp.y = 0;
-        exp_target.x = atan2(x_hat[idx].y, x_hat[idx].x); //!< arg = atan2(imag, real)
+        exp_target.x = atan2(x_hat[idx].y, x_hat[idx].x); //arg = atan2(imag, real)
         exp_target.y = 0;
         y_hat[idx] = cuCmul(mag_comp, gpu_exp(cuCmul(complex1i, exp_target)));
     }
 }
 
 /**
- * Processing magnitudes with mask
+ * \brief Processing random phase array with mask, generating image result
+ * \param mask Input mask
+ * \param y_hat Random phase data with size of dimension
+ * \param image_x Image result
+ * \param image_x_p Image result from previous iteration
+ * \param image_x_comp Complex version of image result in an iteration
+ * \param beta Input beta
+ * \param mode Input mode (int version)
```

Comment:
please explain, what is the int version of the mode (for example, list int value and related mode).